File: sources/salesman/off/agentes/core/templatetags/productos.js
require('doff.template.base', 'Library');
require('agentes.core.models', 'Categoria', 'Producto');

var register = new Library();
function categorias() {
    var categorias = Categoria.objects.filter({ super__isnull: true });
    return { "categorias": categorias };
}
register.inclusion_tag("categorias.html")(categorias);

function novedades(context) {
	var novedades = Producto.objects.order_by('-pk').slice(1, 6);
    return { "novedades": novedades, 'MEDIA_URL': context.get('MEDIA_URL') };
}
register.inclusion_tag("novedades.html", { takes_context: true })(novedades);

publish({
	register: register
});

File: sources/salesman/apps/core/templatetags/productos.py
from django import template
from salesman.apps.core.models import Categoria, Producto

register = template.Library()
@register.inclusion_tag("categorias.html")
def categorias():
    categorias = Categoria.objects.filter(super__isnull = True)
    return { "categorias": categorias }

@register.inclusion_tag("novedades.html", takes_context = True)
def novedades(context):
    novedades = Producto.objects.order_by('-pk')[0:5]
    return { "novedades": novedades, 'MEDIA_URL': context['MEDIA_URL'] }